SDWebImage/SDMemoryCache.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is part of the SDWebImage package.
+ * (c) Olivier Poitrey <rs@dailymotion.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+#import "SDWebImageCompat.h"
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Well. I can remove this by specify nonnull to the most of methods. Unlike the `SDImagCache`. We can block all the optional value from the beginning instead of pass to `id<SDDiskCache>`